With the addition of 043_notebook_scroll.patch in gtk+3.0, it is 
possible to switch tabs with Alt+mouse wheel from anywhere on the 
notebook. This patch prevents deepinvte from interecepting the signal.

Corresponding GTK+ bug: GNOME #145244

Index: deepinvte-0.28.0/src/deepinvte.c
===================================================================
--- deepinvte-0.28.0.orig/src/deepinvte.c	2011-06-05 01:48:33.224480856 +0200
+++ deepinvte-0.28.0/src/deepinvte.c	2011-06-05 01:59:00.267546314 +0200
@@ -11361,6 +11361,13 @@ deepinvte_terminal_scroll(GtkWidget *widget, G
 
 	deepinvte_terminal_read_modifiers (terminal, (GdkEvent*) event);
 
+#if GTK_CHECK_VERSION (2, 90, 8)
+	/* Do not intercept Alt+scroll, let the GtkNotebook handle it to switch tabs.
+	   Requires a fixed GTK+, see https://bugzilla.gnome.org/show_bug.cgi?id=145244 */
+	if (event->state & GDK_MOD1_MASK)
+		return FALSE;
+#endif
+
 	_DEEPINVTE_DEBUG_IF(DEEPINVTE_DEBUG_EVENTS)
 		switch (event->direction) {
 		case GDK_SCROLL_UP:
